index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Responsive Login</title>
+  <link rel="stylesheet" type="text/css" href="css/script.css">
+</head>
+<body>
+  <header>Login</header>
+  <form>
+    <fieldset>
+      <label for="email">Email</label>
+      <br>
+      <input type="email" id="mail" name="email" required autofocus /><br><br>
+
+      <label for="password">Password</label><br>
+      <input type="password" id="psw" name="psw" required minlength="6"><br><br>
```

Comment:
The `for` attribute of label relates to the Element ID -- in your case, that's `psw`.